packages/docs-infra/src/CodeHighlighter/maybeInitialData.ts

```diff
@@ -0,0 +1,122 @@
+import { hasAllVariants } from './hasAllVariants';
+import { Code, VariantExtraFiles, VariantSource } from './types';
+
+export function maybeInitialData(
```

Comment:
Again, JSDocs would help, I am trying to undertand why the function name has maybe, but can't really figure it out :))